doctr/documents/reader.py

```diff
@@ -17,11 +19,55 @@
 AbstractFile = Union[AbstractPath, bytes]
 Bbox = Tuple[float, float, float, float]
 
+def orient_image(img):
+    gray_img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+    gray_img = cv2.medianBlur(gray_img, 5)
+    thresh = cv2.threshold(gray_img, thresh=0, maxval=255, type=cv2.THRESH_BINARY_INV + cv2.THRESH_OTSU)[1]
+
+    # try to merge words in lines
+    (h, w) = img.shape[:2]
+    k_x = max(1, (floor(w / 100)))
+    k_y = max(1, (floor(h / 100)))
+    kernel = cv2.getStructuringElement(cv2.MORPH_RECT, (k_x, k_y))
+    thresh = cv2.dilate(thresh, kernel, iterations=1)
+
+    # extract contours
+    contours, _ = cv2.findContours(thresh, cv2.RETR_LIST, cv2.CHAIN_APPROX_SIMPLE)
+
+    # Sort contours
+    contours = sorted(contours, key=cv2.contourArea, reverse=True)
+
+    n_ct = 50
+    ratio_threshold_for_lines = 5
```

Comment:
Let's make those default values of function's arguments

Reply:
OK will do